<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTask</class>
 <widget class="QDialog" name="EditTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit task</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI Symbol&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/UI/rsc/task.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe UI Symbol</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Source folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceFolderTextBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseSrc">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/UI/rsc/browse.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExclude">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string>Excluded Items...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/UI/rsc/exclude.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/UI/rsc/target.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI Symbol</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination folder</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="targetFolderTextBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseDst">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/UI/rsc/browse.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="robocopySwitchesCheckBox">
     <property name="text">
      <string>Custom copy options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOptions">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Copy options</string>
     </property>
     <widget class="QCheckBox" name="chk_s">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/S</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_m">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/M</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_sec">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/SEC</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_mov">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/MOV</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_a">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/A</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_b">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/B</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_copyall">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/COPYALL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_nocopy">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/NOCOPY</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_mir">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/MIR</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_zb">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/ZB</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_create">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/CREATE</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_purge">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/PURGE</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_move">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/MOVE</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_fat">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/FAT</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_copy">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/COPY:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_lev">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/LEV:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_lev">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>18</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_xo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/XO</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_v">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/V</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_tbd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/TBD</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_np">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/NP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_z">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/Z</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_e">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>80</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/E</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_r">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_r">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>40</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/R:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_w">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>62</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_w">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>62</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/W:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_fft">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>100</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/FFT</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>CopyCommand</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtCmd"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/UI/rsc/okpng.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/UI/rsc/cancel.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
